City-of-Helsinki/helsinki-paatokset
# PP-121 upcoming meetings frontpage

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-121

Created calendar block for frontpage. It has now been configured to only appear on the frontpage but can be changed if needed.

**How to test**

Checkout branch and run `make drush-cim` and make sure the styles have been built.

Create a dummy frontpage and set it as the frontpage from Drupal settings at https://helsinki-paatokset.docker.so/fi/admin/config/system/site-information.

The new block should appear in the page that is set as the frontpage. Check that the styles match the design in desktop and in mobile.

**Things to consider**

The links to the motions and the color line at the top of the box are still placeholders since the actual data doesn't exist yet.
